packages/core/src/asset/request.ts

```diff
@@ -76,6 +76,10 @@ function requestImage<T>(url: string, config: RequestConfig): AssetPromise<T> {
         requestAnimationFrame(() => {
           //@ts-ignore
           resolve(img);
+          img.onload = null;
+          img.onerror = null;
+          img.onabort = null;
+          img = null;
```

Comment:
`img = null;` may not have any effect, after the function is executed, it will be automatically released。

Reply:
I did some tests. `img = null` is really useless.